src/main/java/net/mcreator/element/BaseType.java

```diff
@@ -39,7 +39,8 @@
 	POTION,
 	POTIONEFFECT,
 	FEATURE,
-	TAB;
+	TAB,
+	VILLAGER;

```

Comment:
You can use OTHER here I would say, no need for VILLAGER

Reply:
Yes VILLAGER BaseType is used in `ElementUtil#loadAllVillagerProfessions` so yes it is needed.

Reply:
You should load that from ModElementType, not BaseType

Reply:
All the methods in `ElementUtils` that are loading things from datalists are using `BaseType`

Reply:
There is `getCustomElementsOfType(@Nonnull Workspace workspace, ModElementType<?> type)` with a reason, also don't mark resolved by yourself, I have told you that quite a few times now